docs/source/Appendix_B.md

```diff
@@ -409,7 +409,14 @@ under an appropriate unit class).
 
 #### SCHEMA_CHARACTER_INVALID
 
-**a.**  The specification contains an invalid character for the section in which it appears.  
+**a.** A schema node contains non `name` characters.  

```

Comment:
Add hyphen after "non" throughout this list.

Reply:
Dohe